lib/rialto/etl/configs/wos_to_sparql_statements.rb

```diff
@@ -103,23 +105,39 @@ def lookup_address(addresses, addr_id)
     address = lookup_address(addresses, c['addr_no'])
     person_params = c.slice('orcid_id', 'first_name', 'last_name', 'full_name')
     person_params.merge!(address) if address
-    resolved_person = resolve_entity('person', person_params)
-    new_person = {
-      '@id' => RIALTO_PEOPLE[Digest::MD5.hexdigest("#{c['first_name']} #{c['last_name']}".downcase)],
-      '@type' => [FOAF['Agent'], FOAF['Person']]
-      # TODO: labels and name vcard
-    }
+
+    person = if (resolved_person = resolve_entity('person', person_params))
```

Comment:
How would you feel about extracting this out to a new `resolve_person` method? Useful reduction of code/logic, or unnecessary shifting around?

Reply:
So that would abstract out lines 109-119, right?

Reply:
Yep.